### ISSUE TITLE
Newpipe popup blocking Google play buttons

### ISSUE THREAD

**Issue by mascraft** [TeamNewPipe/NewPipe#7785]
### Checklist

- [x] I am using the latest version - 0.21.16 
- [X] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [X] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [X] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug

Play any video in popup mode
Open Google play store
Open any app's page in the Google play store
try to install/uninstall/open any app in the Google play store

### Actual behavior

Popup players blocks all of the buttons and you can't install/uninstall/open any app via Google play store. (gives a little msg that something is blocking Google play, disable overlay stuff) 

### Expected behavior

You should be able to just install/uninstall/open any app because apps like pure tuber and such which use Newpipe source code have no problem with Google play. 

### Screenshots/Screen recordings

https://user-images.githubusercontent.com/38975749/152360276-b4893943-fe15-4bf1-a738-d05e30ede919.jpg

### Device info

 - Android version/Custom ROM version: android 11/miui 12.5.5.0 global
 - Device model: poco f3 (alioth)

**Comment by mhmdanas** [comment]
I think that's working as expected, you can't interact with system popups when an app is drawing over the screen. I'm not sure how PureTuber does it.

(By the way, don't use PureTuber, it's an illegal fork of NewPipe with no source code released)

**Comment by mascraft** [comment]
Yes I know but they have implemented some stuff that are just too good to pass on.....they somehow made it so Google play buttons work, also they made it so the video will keep being in the background in app switches (when you go to another app it just keeps playing like YouTube vanced). Do you think that we can make a fork like pure tuber without Google login? Like with a better homepage and stuff!

**Comment by mascraft** [comment]
> I think that's working as expected.

I also found this with other apps, other apps let you interact with system pop ups. Every single app I used that has been maintained has this. Maybe it's an feature in higher versions of android!

**Comment by mhmdanas** [comment]
>  also they made it so the video will keep being in the background in app switches (when you go to another app it just keeps playing like YouTube vanced).

If I understood you correctly, you want "Settings > Video and audio > Minimize on app switch". Is that what you want?

> I also found this with other apps, other apps let you interact with system pop ups.

Hmm, that's strange.

**Comment by barackenkumpel** [comment]
I think why you are able to interact with system dialogs using other apps is because they uses the picture-in-picture support of android 8 and above (don't ask me about details I'm just a kid. I have this from here: [https://developer.android.com/guide/topics/ui/picture-in-picture](https://developer.android.com/guide/topics/ui/picture-in-picture)). NewPipe uses the SYSTEM_ALERT_WINDOW AppOp (in system settings called "Display over other apps"), so the dialogs are disabled. But since PiP is not supported for all people's android versions there are certainly some reasons why the people developing this app went this way.

**Comment by mhmdanas** [comment]
@barackenkumpel that's very possible, I totally forgot about PiP.

**Comment by Wilburampton** [comment]
Thank a lot

**Comment by mkikets99** [comment]
Hey, its already 0.22.2...
Is there any way to make a version with PiP?
